### PR TITLE
22966 move journey api into bb sdk

### PR DESCRIPTION
**Pull request by nnorbert**
Move Journey API into the Building Blocks SDK package.
Only Journey service was updated, Payments and Dashboard services are still using the Journey API calls from the old package. That will be updated once the new Journey API package was published.

### CODE REVIEW

File: apps/journey/libraries/building-block-sdk.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  const res = await fetch(
+    new URL(
+      path,
+      process.env.NEXT_PUBLIC_JOURNEY_SERVICE_ENTRY_POINT as string,
+    ),
+    { headers: { cookie: cookieHeader } },
+  );
+  const { token } = await res.json();
+  return token;
+}
```
Comment (peschina) [inline apps/journey/libraries/building-block-sdk.ts:75]: Can you please add EOF?

**Review by peschina — APPROVED** [review]
LGTM, one minor comment

File: apps/journey/libraries/building-block-sdk.ts (hunk 1 above)
Comment (nnorbert) [reply apps/journey/libraries/building-block-sdk.ts]: Done!

**Review by alfonsograziano — DISMISSED** [review]
LGTM

**Review by peschina — APPROVED** [review]

**Pull request merged**